mods/optifine.json

```diff
@@ -0,0 +1,4775 @@
+{
+    "format": 1,
+    "name": "OptiFine",
+    "authors": ["sp614x"],
+    "disc": "Minecraft optimization mod",
+    "discLong": "This mod adds support for HD textures and a lot of options for better looks and performance. Doubling the FPS is common.",
+    "homepage": "http://optifine.net",
+    "license": {
+        "type": "custom",
+        "redistrubution": "false",
+        "text": "The mod OptiFine is Copyright © 2013 by sp614x and the intellectual property of the author. It may be not be reproduced under any circumstances except for personal, private use as long as it remains in its unaltered, unedited form. It may not be placed on any web site or otherwise distributed publicly without advance written permission. Use of this mod on any other website or as a part of any public display is strictly prohibited and a violation of copyright."
+    },
+    "versions": [
+        {
+            "name": "LagSpikeOfDeath_1.7.2.zip",
+            "mcvsn": ["beta_TODO"],
+            "hash": { "type": "sha256", "digest": "83fe6b5294161eb33b2f18f32a4af5077fd493a6047b1278ac448475d1fe159d"},
+            "urls": [
+                { "type": "archived", "url": "TODO", "desc": "TODO" },
+                { "type": "original", "url": "TODO", "desc": "TODO" },
```

Comment:
Same here. If there is no original direct download, this entry should be excluded from the list of URLs.